apps/gitness/src/App.tsx

```diff
@@ -49,6 +51,20 @@ export default function App() {
           path: 'pull-requests',
           element: <PullRequestListPage />
         },
+        {
+          path: 'pull-requests/:pullRequestId',
+          element: <PullRequestLayout />,
+          children: [
+            {
+              index: true,
+              element: <Navigate to="commits" />
```

Comment:
I assume this is just for testing, by default the `Conversation` tab should be opened?

Reply:
yes, we can change later on